### DISCUSSION TITLE
how to host without docker

### DISCUSSION THREAD

**Discussion by nipunattri1** [fonoster/yaps#227] [Q&A] [answered]
i want to host it without docker how to do it?

**Comment by psanders** [comment, marked as answer]
> i want to host it without docker how to do it?

This is currently not possible.

What makes you want to run without Docker?

**Reply by nipunattri1** [reply]
Well I read that it's made with node js and I know about a hosting site called deta.sh and i want to host it there and some of my local services don't support docker ( yeah I know this is odd)

**Reply by psanders** [reply]
I see. You would have to run your instance on a full cloud provider (GCP, Azure, DO, AWS, etc.)

And just clarify you only need docker on your serverside, not on the clients.

**Reply by nipunattri1** [reply]
Hm., Okay then, will see about that 😐

**Reply by psanders** [reply]
@nipunattri1 

Sounds good. Just remember that you can use the [SaaS option](https://console.fonoster.io/) until you figure out how to run your own instance.
